### ISSUE TITLE
Add XGBoost end model

### ISSUE THREAD

**Issue by ajratner** [HazyResearch/snorkel#585]
https://en.wikipedia.org/wiki/Xgboost

Has Python / scikit learn bindings, can probably use this to put in with noise aware loss pretty quickly

**Comment by henryre** [comment]
Weird that we both thought of this at the same time. Looks like they use LogitBoost by default, and allow custom losses:

https://github.com/dmlc/xgboost/blob/master/demo/guide-python/custom_objective.py

**Comment by ajratner** [comment]
Nice :D
